components/Search/forms/SearchResidentsForm.tsx

```diff
@@ -40,7 +40,7 @@ const SearchResidentsForm = ({
       <div className="govuk-grid-row">
         <div className="govuk-grid-column-one-half">
           <TextInput
```

Comment:
i think we need to tweak the classes on these a tiny bit more to get the spacing just right. copying the [example from the design system](https://design-system.hackney.gov.uk/components/input), i think the output should look like:

```
<div class="govuk-form-group lbh-form-group">
    <label for="first_name" class="govuk-label lbh-label">First name </label>
    <input id="first_name" data-testid="first_name" name="first_name" type="text" class="govuk-input lbh-input">
</div>
```

Reply:
Bit confused what you mean here. These are all defined in the TextInput.tsx component. Do you mean you'd like changes to that?

Reply:
if that's the easiest way, i think so, yes